ui/src/app/store/vlan/utils.ts

```diff
@@ -0,0 +1,19 @@
+import type { VLAN } from "app/store/vlan/types";
+
+/**
+ * Get the VLAN display text.
+ * @param vlan - A VLAN.
+ * @return The VLAN display text.
+ */
+export const getVLANDisplay = (vlan: VLAN | null): string | null => {
+  if (!vlan) {
+    return null;
+  }
+  if (vlan.vid === 0) {
+    return "untagged";
```

Comment:
Is the vid related to the tag?

Reply:
Can you add a test for this so it's clear is expected

Reply:
In MAAS vid 0 is used for untagged vlans (https://git.launchpad.net/maas/tree/src/maasserver/models/vlan.py#n40).

I've added a test, and because this is a special case I've added an ENUM to the VLAN type for vid 0.